### DISCUSSION TITLE
Cyfrin / foundry-nft-f23

### DISCUSSION THREAD

**Discussion by DCVglobalnetwork** [Cyfrin/foundry-full-course-cu#1694] [General]
HI 
all went well until I started to deploy but I have done it before and worked with anvil but not with other testnet 
yes I put in .env file PRIVATE-KEY and SEPOLIA-URL
if anyone knows what is the issuse please 

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/105791829/fe489d89-762c-44b2-8fdc-0708a509e61f)

**Comment by EngrPips** [comment]
Hello @DCVglobalnetwork, Do you have a value for `etherscan-api-key` in your `.env` file?
